### ISSUE TITLE
Error while installing 7.4RC

### ISSUE THREAD

**Issue by shivammathur** [mlocati/powershell-phpmanager#35]
While installing PHP 7.4RC it gives this error
```
Failed to recognize VCVersion from Visual C++ 2017
At C:\Users\runneradmin\Documents\WindowsPowerShell\Modules\PhpManager\1.16.1.201\private\PhpVersion.ps1:284 char:27
+ ...   default { throw "Failed to recognize VCVersion from Visual C++ $vcY ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (Failed to recog...Visual C++ 2017:String) [], RuntimeException
    + FullyQualifiedErrorId : Failed to recognize VCVersion from Visual C++ 2017
```
This is because in PhpVersion.ps1 the switch to check `$vcYear` only has 2019 and it fails for 2017.
https://github.com/mlocati/powershell-phpmanager/blob/cbda459ab7d32cea30dfbf4b7d4b1e834f7cc386/PhpManager/private/PhpVersion.ps1#L282-L285

**Comment by shivammathur** [comment]
@mlocati Here is a pull request to fix this #36.

**Comment by mlocati** [comment]
Fixed in #36
